packages/venia-ui/lib/components/CheckoutPage/ItemsReview/showAllItemsFooter.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import { mergeClasses } from '../../../classify';
+import Button from '../../Button';
+
+import defaultClasses from './itemsReview.css';
```

Comment:
This component should have its own CSS file. Also, the classnames should be as simple as possible here: `root` and `button`.